### ISSUE TITLE
bug in travel-list project

### ISSUE THREAD

**Issue by deardaveed** [jonasschmedtmann/ultimate-react-course#18]
the CSS for packing-list doesn't work properly.

related to .list ul overflow: scroll I think

**Comment by WaqarAhmad321** [comment]
Press F3 in vs code, search overflow in CSS file and to auto. It will fix the issue.

**Comment by deardaveed** [comment]
thanks! and for anyone else reading, this also needs to be fixed in the usePopcorn starter css file (change overflow from scroll to auto)

**Comment by WaqarAhmad321** [comment]
Yes, you're right it also need to be fixed in worldwise and the wild oasis starter files.

**Comment by Syedsafwan24** [comment]
> Press F3 in vs code, search overflow in CSS file and to auto. It will fix the issue.

Thanks You
